gordo_components/builder/local_build.py

```diff
@@ -75,6 +74,8 @@ def local_build(
     Iterable[Tuple[Union[BaseEstimator, None], dict]]
         A generator yielding tuples of models and their metadata.
     """
+    from gordo_components.builder.mlflow_utils import mlflow_context, log_metadata
```

Comment:
This is a new style, any reason why we are doing this only(?) here and not everywhere? Are these imports particularly problematic?

Reply:
OH ya! The test won't even run if it's where it was... :man_shrugging: ML flow confusion.